packages/@aws-cdk/aws-ecs/test/test.container-definition.ts

```diff
@@ -408,38 +408,80 @@ export = {
 
     // WHEN
     taskDefinition.addContainer('Container', {
-        image: ecs.ContainerImage.fromRegistry('user-x/my-app', {
-            credentials: repoCreds
-        }),
-        memoryLimitMiB: 2048,
+      image: ecs.ContainerImage.fromRegistry('user-x/my-app', {
+        credentials: repoCreds
+      }),
+      memoryLimitMiB: 2048,
     });
 
     // THEN
     expect(stack).to(haveResourceLike('AWS::ECS::TaskDefinition', {
       ContainerDefinitions: [
         {
           Image: 'user-x/my-app',
-            RepositoryCredentials: {
-              CredentialsParameter: mySecretArn
-            },
+          RepositoryCredentials: {
+            CredentialsParameter: mySecretArn
+          },
         }
       ]
     }));
 
     expect(stack).to(haveResourceLike('AWS::IAM::Policy', {
       PolicyDocument: {
-          Statement: [
-              {
-                  Action: "secretsmanager:GetSecretValue",
-                  Effect: "Allow",
-                  Resource: mySecretArn
-              }
-          ]
+        Statement: [
+          {
+            Action: "secretsmanager:GetSecretValue",
+            Effect: "Allow",
+            Resource: mySecretArn
+          }
+        ]
       }
     }));
 
     test.done();
   },
 
+  'can specify linux parameters'(test: Test) {
+    // GIVEN
+    const stack = new cdk.Stack();
+    const taskDefinition = new ecs.Ec2TaskDefinition(stack, 'TaskDef');
+
+    const linuxParameters = new ecs.LinuxParameters(stack, 'LinuxParameters', {
+      initProcessEnabled: true,
+      sharedMemorySize: 1024,
+    });
+
+    linuxParameters.addCapabilities(ecs.Capability.All);
```

Comment:
Can you add a test where we do `linuxParameters.addCapabilities()` *after* passing them to `addContainer()`? Since the object is mutable, I'd like to confirm that order doesn't matter, and that this stays true even if we refactor in the future.